prow/plugins/trigger/pull-request.go

```diff
@@ -137,7 +137,21 @@ func buildAllIfTrusted(c client, trigger *plugins.Trigger, pr github.PullRequest
 }
 
 func welcomeMsg(ghc githubClient, trigger *plugins.Trigger, pr github.PullRequest) error {
-	commentTemplate := `Hi @%s. Thanks for your PR.
+	var commentTemplate string
+	if trigger.IgnoreOkToTest {
+		commentTemplate = `Hi @%s. Thanks for your PR.
```

Comment:
This template requires different parameters from the original template, but is still created with the original parameters on line 181 so this won't format properly.